src/Compilers/CSharp/Portable/Lowering/ClosureConversion/LambdaCapturedVariable.cs

```diff
@@ -59,19 +59,14 @@ private static string GetCapturedVariableFieldName(Symbol variable, ref int uniq
 
             if (variable is LocalSymbol local)
             {
-                if (local.SynthesizedKind == SynthesizedLocalKind.LambdaDisplayClass)
+                switch (local.SynthesizedKind)
```

Comment:
This commit was purely invoking the "convert to switch expression" refactoring, which I did to improve future maintainability here.